apps/sms/js/thread_list_ui.js

```diff
@@ -393,17 +394,24 @@ var ThreadListUI = {
     }
   },
 
-  markReadUnread: function thlui_markReadUnread(selected, isRead) {
-    selected.forEach((id) => {
+  markReadUnreadUI: function thlui_markReadUnreadUI(selected, isRead) {
+    function filterIds(id) {
```

Comment:
The function name does not give a lot of information here; we already know it's a filter because we use it in the "filter" function. Moreover it does an action but we don't know it from the name.

Instead you can use something like: "markVisuallyAndFilterMarkable"